### PR TITLE
Updated berat info, including wiki link

### PR DESCRIPTION
**Pull request by AntoniaCoote**

### CODE REVIEW

File: web_source/cities/europe/albania/berat.md [hunk 1]
```diff
@@ -4,12 +4,14 @@ Country: Albania (Europe)
...
 
-FIXME
+Berati
 
 ### What we would like to see there
 
-FIXME
+Berat Castle, and to admire the view to the city. 
+
+![Berat Castle](https://en.wikipedia.org/wiki/Berat#/media/File:Berat_citadel.jpg)
```
Comment (eccr) [inline web_source/cities/europe/albania/berat.md]: I think this link might be broken.

**Review by eccr — CHANGES_REQUESTED** [review]
Great information - just one possibly broken link to fix I think.

File: web_source/cities/europe/albania/berat.md (hunk 1 above)
Comment (AntoniaCoote) [reply web_source/cities/europe/albania/berat.md]: Thank you for spotting the broken link, I have hopefully updated with one that works! Please review again.

Comment (eccr) [reply web_source/cities/europe/albania/berat.md]: Great image, thanks!

**Review by eccr — APPROVED** [review]
Lovely new image, thank you!

**Pull request merged**